packages/workbox-core/src/controllers/WorkboxCore.js

```diff
@@ -0,0 +1,20 @@
+import assert from '../utils/assert';
+import logHelper from '../utils/logHelper';
+
+export default class WorkboxCore {
+  get INTERNAL() {
+    if (this._internal) {
+      return this._internal;
+    }
+
+    this._internal = {
+      logHelper,
+    };
+
+    if (process.env.NODE_ENV !== 'production') {
+      this._internal.assert = assert;
+    }
```

Comment:
So our code-stripping is wise enough to exclude assertions like this one from the final builds if the right environment is specified?